client/my-sites/email/titan-set-up-thank-you/index.tsx

```diff
@@ -73,7 +74,17 @@ const TitanSetUpThankYou = ( props: TitanSetUpThankYouProps ): JSX.Element => {
 					"Access your email on the go with Titan's Android and iOS apps."
 				),
 				stepCta: (
-					<FullWidthButton href={ titanControlPanelUrl } target="_blank">
+					<FullWidthButton
```

Comment:
We're not triggering an event for the "Go to Inbox" button above -- please add that.